float64 north
float64 east
float32 depth
bool mission_enable
bool depth_only
uint16 waypoint_number
